src/utils/data-manager.js

```diff
@@ -158,6 +158,10 @@ export default class DataManager {
     this.currentPage = 0;
   }
 
+  setSearchInitialText(searchInitialText) {
+    this.searchInitialText = searchInitialText;
+  }
```

Comment:
Why you didn't use searchText.

Reply:
Hi Mehmet, just modifying `searchText` didn't seem to be enough: I also need to notify `onSearchChange` - which then set's `searchText` on the `dataManager`. I did not want to interfere with this. If you say it's save to use, I can change it, but I did feel like it is cleaner when `searchText` is only modified from one place at a time.